### DISCUSSION TITLE
Stable Kanidm Version with generate certificate available?

### DISCUSSION THREAD

**Discussion by SamTV12345** [kanidm/kanidm#1781] [Q&A] [answered]
### I did this

I followed the quick start guide. Every step until the generate-certificate worked without a problem. Do you have a stable 

### I expected the following

It generates a certificate for evaluation purposes

### Kanidm version (and git commit)

[kanidm(d) --version]

### Operating System / Version

Darwin Mac Apple 

### Any other comments

I executed with --help and it outputs only this:

USAGE:
    kanidmd <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    configtest         Test the IDM Server configuration, without starting network listeners
    database           Database maintenance, backups, restoration etc
    db-scan            Inspect the internal content of the database datastructures
    domain             Change domain settings
    healthcheck        Load the server config and check services are listening
    help               Print this message or the help of the given subcommand(s)
    recover-account    Recover an account's password
    server             Start the IDM Server
    version            Print the program version and exit
    
There is unfortunately no option named generate-certificate available.

**Comment by Firstyear** [comment, marked as answer]
You are reading the git-master docs. The next release will be on july 1st and will update the stable book and release with this new feature.
